app/email.go

```diff
@@ -101,8 +101,8 @@ func SendVerifyEmail(userId, userEmail, locale, siteURL string) *model.AppError
 func SendSignInChangeEmail(email, method, locale, siteURL string) *model.AppError {
 	T := utils.GetUserTranslations(locale)
 
-	subject := T("api.templates.singin_change_email.subject",
-		map[string]interface{}{"SiteName": utils.ClientCfg["SiteName"]})
+	subject := fmt.Sprintf("[%v] %v", utils.Cfg.TeamSettings.SiteName, T("api.templates.singin_change_email.subject",
```

Comment:
So the string is `You updated your sign-in method on {{ .SiteName }}` right?

I guess that's fine to leave it this way but it feels weird having the same variable being used twice